Backup/restore-to-alternate-server.md

```diff
@@ -1,13 +1,14 @@
 {{{
   "title": "Restore to Alternate Server",
-  "date": "08-13-2018",
+  "date": "02-27-2019",
   "author": "Dan Benton",
   "attachments": [],
   "related-products" : [],
   "contentIsHTML": false,
   "sticky": false
 }}}
 
+# **This document is deprecated, please see our [restore document](./restores.md) for a much simpler, UI driven way to restore to alternate servers.**
```

Comment:
Please use `###` as the largest header size.  You also don't need bold within a header.